processors/piifilterprocessor/processor.go

```diff
@@ -176,26 +180,33 @@ func (p *piiFilterProcessor) processComplexData(attrs pdata.Span) {
 				var err error
 				dataType, err = getDataTypeFromContentType(typeValue.StringVal())
 				if err != nil {
-					p.logger.Sugar().Debugf("could not parse media type %q: %v", typeValue.StringVal(), err)
+					p.logger.Debug(
+						fmt.Sprintf("could not parse media type %q", typeValue.StringVal()),
```

Comment:
why a sprintf rather than something like zap.String("media_type", typeValue) ?

Reply:
I don't think it will improve the logging experience as this field is very specific to this error and indexing it might be too much.